update.php

```diff
@@ -92,23 +92,10 @@ function update_1191()
 	DI::config()->set('system', 'maintenance', 1);
 
 	if (Addon::isEnabled('forumlist')) {
-		$addon = 'forumlist';
-		$addons = DI::config()->get('system', 'addon');
-		$addons_arr = [];
-
-		if ($addons) {
-			$addons_arr = explode(",", str_replace(" ", "", $addons));
-
-			$idx = array_search($addon, $addons_arr);
-			if ($idx !== false) {
-				unset($addons_arr[$idx]);
-				//delete forumlist manually from addon and hook table
-				// since Addon::uninstall() don't work here
-				q("DELETE FROM `addon` WHERE `name` = 'forumlist' ");
-				q("DELETE FROM `hook` WHERE `file` = 'addon/forumlist/forumlist.php' ");
-				DI::config()->set('system', 'addon', implode(", ", $addons_arr));
-			}
-		}
+		//delete forumlist manually from addon and hook table
+		// since Addon::uninstall() don't work here
```

Comment:
Why doesn’t it work here?

Reply:
I don't know. This comment had been there before, so I didn't wanted to challenge my luck.

Reply:
These two lines are literally what `Addon::uninstall` does after you removed `Addon::saveEnabledList`, please replace it with this call.

Reply:
But the uninstall does some additional magic. I would like to stay on the safe side here.

Reply:
It just runs the addon uninstall function if it exists, that's it. It used to do more magic, but not anymore thanks to you.